metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -2687,8 +2687,12 @@ class MetalsLanguageServer(
       onError = {
         case e @ (_: ParseException | _: TokenizeException) =>
           scribe.error(e.toString)
-        case e: InvalidJarException =>
+        case e: IndexingExceptions.InvalidJarException =>
           scribe.warn(s"invalid jar: ${e.path}")
+        case e: IndexingExceptions.PathIndexingException =>
+          scribe.warn(s"issues while parsing: ${e.path}")
+        case e: IndexingExceptions.InvalidSymbolException =>
+          scribe.warn(s"searching for `${e.symbol}` failed")
```

Comment:
Maybe it's worth to log `e.underlying` exception?

Reply:
Fixed!